### ISSUE TITLE
Missing cu version in nav2018 Artifact Manifest

### ISSUE THREAD

**Issue by ChrisBlankDe** [microsoft/navcontainerhelper#2853]
**Describe the issue**
Some of the newest NAV 2018 Artifact (starting with cu53) are missing the cu version in the artifact manifest.json.

**Scripts used to reproduce**
```posh
Get-NavArtifactUrl -nav 2018 -cu 49 -country w1 | %{Download-Artifacts -artifactUrl $_} | Join-Path -ChildPath 'manifest.json'| gi| gc | ConvertFrom-Json | select Version,cu #ok
Get-NavArtifactUrl -nav 2018 -cu 50 -country w1 | %{Download-Artifacts -artifactUrl $_} | Join-Path -ChildPath 'manifest.json'| gi| gc | ConvertFrom-Json | select Version,cu #ok
Get-NavArtifactUrl -nav 2018 -cu 51 -country w1 | %{Download-Artifacts -artifactUrl $_} | Join-Path -ChildPath 'manifest.json'| gi| gc | ConvertFrom-Json | select Version,cu #ok
Get-NavArtifactUrl -nav 2018 -cu 52 -country w1 | %{Download-Artifacts -artifactUrl $_} | Join-Path -ChildPath 'manifest.json'| gi| gc | ConvertFrom-Json | select Version,cu #ok
Get-NavArtifactUrl -nav 2018 -cu 53 -country w1 | %{Download-Artifacts -artifactUrl $_} | Join-Path -ChildPath 'manifest.json'| gi| gc | ConvertFrom-Json | select Version,cu
Get-NavArtifactUrl -nav 2018 -cu 54 -country w1 | %{Download-Artifacts -artifactUrl $_} | Join-Path -ChildPath 'manifest.json'| gi| gc | ConvertFrom-Json | select Version,cu
Get-NavArtifactUrl -nav 2018 -cu 55 -country w1 | %{Download-Artifacts -artifactUrl $_} | Join-Path -ChildPath 'manifest.json'| gi| gc | ConvertFrom-Json | select Version,cu
Get-NavArtifactUrl -nav 2018 -cu 56 -country w1 | %{Download-Artifacts -artifactUrl $_} | Join-Path -ChildPath 'manifest.json'| gi| gc | ConvertFrom-Json | select Version,cu
Get-NavArtifactUrl -nav 2018 -cu 57 -country w1 | %{Download-Artifacts -artifactUrl $_} | Join-Path -ChildPath 'manifest.json'| gi| gc | ConvertFrom-Json | select Version,cu
Get-NavArtifactUrl -nav 2018 -cu 58 -country w1 | %{Download-Artifacts -artifactUrl $_} | Join-Path -ChildPath 'manifest.json'| gi| gc | ConvertFrom-Json | select Version,cu
Get-NavArtifactUrl -nav 2018 -cu 59 -country w1 | %{Download-Artifacts -artifactUrl $_} | Join-Path -ChildPath 'manifest.json'| gi| gc | ConvertFrom-Json | select Version,cu
Get-NavArtifactUrl -nav 2018 -cu 60 -country w1 | %{Download-Artifacts -artifactUrl $_} | Join-Path -ChildPath 'manifest.json'| gi| gc | ConvertFrom-Json | select Version,cu

```

**Full output of scripts**
```
version      cu  
-------      --  
11.0.48302.0 cu49
11.0.48415.0 cu50
11.0.48514.0 cu51
11.0.48680.0 cu52
11.0.48794.0 cu  
11.0.48962.0 cu  
11.0.49061.0 cu  
11.0.49175.0 cu  
11.0.49240.0 cu  
11.0.49345.0 cu  
11.0.49497.0 cu  
11.0.49618.0 cu  
```

**Additional context**
- does it happen all the time? yes
- did it use to work? till cu52

**Comment by freddydk** [comment]
That is probably the version from which the CUs were autogenerated...
Will see how I can fix these

**Comment by freddydk** [comment]
Easiest thing might be to patch this after the download:-)

**Comment by freddydk** [comment]
Especially since NAV 2018 is history and BC doesn't have a CU property

**Comment by ChrisBlankDe** [comment]
> Easiest thing might be to patch this after the download:-)

lgtm. should i create a pr?

**Comment by freddydk** [comment]
You are welcome to:-)

**Comment by freddydk** [comment]
Fixed in 4.0.14

**Comment by freddydk** [comment]
Thanks
